swarm/netsim/http.go

```diff
@@ -17,45 +17,34 @@
 package netsim
 
 import (
-	"context"
 	"fmt"
 	"net/http"
 
 	"github.com/ethereum/go-ethereum/log"
 	"github.com/ethereum/go-ethereum/p2p/simulations"
 )
 
-func (s *Simulation) initHTTPServer(opts *SimulationOptions) {
+// Package defaults.
+var (
+	DefaultHTTPSimPort = "8888"
```

Comment:
Maybe we can just use the address as it is used in http.Server, not just the port.

Like this:

```
DefaultHTTPAddr = ":8888"
```

Reply:
hmmm do you really think this is a good idea? I know it makes the API cleaner, but isn't it most probably that users would forget to add ":"?

Reply:
I think that making it more consistent with stdlib's net and net/http would be more intuitive to users. By calling it addr instead of port is more clear, also the DefaultHTTPAddr suggests the required format.